packages/shared/components/Alert.svelte

```diff
@@ -55,9 +55,12 @@
         icon={TOAST_STYLE[type].icon}
         classes="fill-current {TOAST_STYLE[type].iconColor}"
     />
-    <div class="flex flex-auto flex-col px-4">
-        <Text fontWeight={FontWeight.semibold} class="flex text-13 {TOAST_STYLE[type].messageColor}">{message}</Text>
-    </div>
+    <message-container class="w-full flex flex-auto flex-col px-4">
+        <Text
+            fontWeight={FontWeight.semibold}
+            class="flex text-13 text-ellipsis overflow-hidden {TOAST_STYLE[type].messageColor}">{message}</Text
```

Comment:
Not sure this text-ellipsis and overflow-hidden are working. I am testing creating a notification with a very large text and its being cut. I would not add that at all because we actually should never cut a message, and instead add a `break-words` class so the message is always displayed

Reply:
The same should be added not only to alerts but to any type of app notification message